classes/class-wc-connect-api-constants.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * A class for constants used by the API.
+ */
+class WC_Connect_API_Constants {
```

Comment:
Yea, I was thinking this constants file either to be a "file for all WCS&T constants" or "file for connect client/API related constants", thus the naming.

Instead of creating a general constants file, I like the `WC_Connect_WCST_To_WCShipping_Migration_State_Enum` suggestion below. I will update it with an enum class.

Reply:
Updated in 85a7e614